Source/Core/Core/NetPlayClient.cpp

```diff
@@ -147,8 +147,12 @@ NetPlayClient::NetPlayClient(const std::string& address, const u16 port, NetPlay
     m_client->mtu = std::min(m_client->mtu, NetPlay::MAX_ENET_MTU);
 
     ENetAddress addr;
-    enet_address_set_host(&addr, address.c_str());
+    if (auto parsed_addr = Common::ENet::AddressFromString(address))
+    {
+      addr.host = *parsed_addr;
+    }
```

Comment:
```suggestion
    addr.host = Common::ENet::AddressFromString(address).value_or({});
```
The current code leaves host uninitialized on parse failure.